decidim-meetings/app/controllers/decidim/meetings/admin/meeting_closes_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+module Decidim
+  module Meetings
+    module Admin
+      # This controller allows an admin to manage meetings from a Participatory Process
+      class MeetingClosesController < Admin::ApplicationController
+        helper ResourceHelper
+
+        def edit
```

Comment:
IMO this should be new/create instead of edit/update, since what you are dealing with here is the event of closing a meeting, not the meeting itself, so you are creating a "close this meeting" event, not editing the meeting